libs/SalesforceSDK/AndroidManifest.xml

```diff
@@ -32,6 +32,7 @@
             android:theme="@style/SalesforceSDK"
             android:launchMode="singleTask"
             android:windowSoftInputMode="adjustResize"
+            android:configChanges="orientation|screenLayout|uiMode|screenSize|smallestScreenSize"
```

Comment:
This allows for rotation, foldable changes, multitasking, etc without the webview reloading.